clang/include/clang/Sema/Sema.h

```diff
@@ -7628,6 +7628,10 @@ class Sema final {
   /// CheckOverrideControl - Check C++11 override control semantics.
   void CheckOverrideControl(NamedDecl *D);
 
+  /// CheckVirtualSYCLAddIRAttributesFunctionAttr - Check and diagnose if a
+  /// SYCLAddIRAttributesFunctionAttr is attached to a virtual member function.
+  void CheckVirtualSYCLAddIRAttributesFunctionAttr(NamedDecl *D);
```

Comment:
```suggestion
  void CheckVirtualSYCLAddIRAttributesFunctionAttr(const NamedDecl *D);
```
If we can make this const correct, let's do so. If takes more changes than this (and the definition), don't worry about it; no need to make viral changes if that's what's needed to make this const correct.

Reply:
That's an oversight from my side. No reason for it not to be.